pkgs/tools/system/netdata/default.nix

```
@@ -59,6 +60,20 @@ in stdenv.mkDerivation rec {
     # Avoid build-only inputs in closure leaked by configure command:
     #   https://github.com/NixOS/nixpkgs/issues/175693#issuecomment-1143344162
     ./skip-CONFIGURE_COMMAND.patch
+
+    (fetchpatch {
+      # Fix stream authentication: https://github.com/netdata/netdata/pull/14063
+      name = "fix-stream-authentication";
```

Having `.patch` in the name was right. I don't like having the redundant title there, maybe you can think of something better to go beside the link that isn't just the title?

```suggestion
      # https://github.com/netdata/netdata/pull/14063
      name = "fix-stream-authentication.patch";
```